build-tools/create-pkg/create-pkg.targets

```diff
@@ -45,6 +45,10 @@
         SourceFiles="@(MonoDocFiles)"
         DestinationFolder="$(XAFrameworkDir)\lib\monodoc"
     />
+    <Copy
+        SourceFiles="@(ThirdPartyNotice)"
```

Comment:
I think we could add this source to the existing version and license file copy task below to save a few lines if desirable, for example:
```diff
-        SourceFiles="@(VersionFiles);$(PkgLicenseSrcEn);"
+        SourceFiles="@(VersionFiles);$(PkgLicenseSrcEn);@(ThirdPartyNotice)"
```